src/SalesforceFunctions.php

```diff
@@ -17,17 +17,17 @@ class SalesforceFunctions
     /**
      * @var string
      */
-    const apiVersion = "v48.0";
+    private const apiVersion = "v48.0";
```

Comment:
I see a `private` has snuck in here. Since this is the default value and ideally used internally, this could stay?